cmd/goflow2/main.go

```diff
@@ -193,6 +193,36 @@ func main() {
 			numSockets = 1
 		}
 
+		numWorkers := 0
+		if listenAddrUrl.Query().Has("workers") {
+			if numWorkersTmp, err := strconv.ParseUint(listenAddrUrl.Query().Get("workers"), 10, 64); err != nil {
+				log.Fatal(err)
+			} else {
+				numWorkers = int(numWorkersTmp)
+			}
+		}
+		if numWorkers == 0 {
+			numWorkers = numSockets * 2
+		}
+
+		isBlocking := false
+		if listenAddrUrl.Query().Has("blocking") {
+			if isBlocking, err = strconv.ParseBool(listenAddrUrl.Query().Get("blocking")); err != nil {
+				log.Fatal(err)
+			}
+		}
+
+		queueSize := 0
```

Comment:
small nits `var queueSize int`

and `var isBlocking bool`

Reply:
Fixed